### ISSUE TITLE
Bump CompanyInfo's minorversion to 3

### ISSUE THREAD

**Issue by drewish** [ruckus/quickbooks-ruby#371]
I spent a while wondering why the SubscriptionStatus didn't match what QBO showed on: https://developer.intuit.com/docs/0100_quickbooks_online/0100_essentials/000800_manage_your_apps/quickbooks_online_subscription_states

Then I found this in https://developer.intuit.com/docs/api/accounting/companyinfo
<img width="509" alt="screen shot 2017-03-31 at 4 31 26 pm" src="https://cloud.githubusercontent.com/assets/68750/24572075/f9a82bce-1632-11e7-9f52-0adfe418ebd6.png">

And at that point, little hint "returned in the SubscriptionStatus attribute of the CompanyInfo resource (with query parameter minorversion=3)" made a lot more sense ;)

**Comment by minimul** [comment]
Nice find. It is probably worth considering running with the latest minorversion globally by default. These minorversions usually just add new features so the chance of breakage is limited. There are fixes like with the Reports API in mv=3 (and other fixes) so users doing workarounds there might have breakage. Then again maybe there is not enough features in these minorversions to bother making the default.

**Comment by drewish** [comment]
Yeah might be worth moving the minor version code out of the Item service and into ServiceCrud where other classes could use it.
